.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["react-native"]
+  "presets": ["react-native"],
+  "plugins": [
+    ["module-resolver", {
```

Comment:
I am interested in knowing why we need a module resolver.

Reply:
The tests fail, producing an `Cannot find module 'react-dom/server' from 'react-apollo.umd.js'` message on the console and on travis. When I searched for solutions, I found that as a fix. I am open to a sleeker alternative.